src/core/MOM.F90

```diff
@@ -2509,6 +2513,13 @@ subroutine initialize_MOM(Time, Time_init, param_file, dirs, CS, &
   call get_param(param_file, "MOM", "DEBUG_OBCS", CS%debug_OBCs, &
                  "If true, write out verbose debugging data about OBCs.", &
                  default=.false., debuggingParam=.true., do_not_log=(number_of_OBC_segments<=0))
+  call get_param(param_file, "MOM", "ENABLE_BUGS_BY_DEFAULT", enable_bugs, &
+                 "If true, the defaults for certain recently added bug-fix flags are set to "//&
+                 "recreate the bugs so that the code can be moved forward without changing "//&
+                 "answers for existing configurations.  The defaults for groups of bug-fix flags "//&
+                 "are periodcially changed to correct the bug, at which point this parameter "//&
+                 "will no longer be used to set their default.  Setting this to false means that "//&
+                 "bugs are only used if they are actively selected.", default=.true.)
```

Comment:
I would more clearly state that if this is set to `false` answers may change when code is updated due to newly found bugs. I think that can be inferred from the description of the `true` option, but because answer changes are not expected with code updates I would suggest a clearer disclaimer here that with this parameter `false` that rule will no longer apply.